### PR TITLE
options_cop: formula -> formulae

### PR DESCRIPTION
**Pull request by commitay**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/master/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/homebrew/pull/49031).
- [x] Have you successfully run `brew style` with your changes locally?
- [x] Have you successfully run `brew tests` with your changes locally?

-----
cc @ilovezfs

### CODE REVIEW

File: Library/Homebrew/rubocops/options_cop.rb [hunk 1]
```diff
@@ -46,8 +46,8 @@ def audit_formula(_node, _class_node, _parent_class_node, body_node)
 
     module NewFormulaAudit
       class Options < FormulaCop
-        DEP_OPTION = "New Formula should not use `deprecated_option`".freeze
-        OPTION = "Formula should not have an `option`".freeze
+        DEP_OPTION = "New Formulae should not use `deprecated_option`".freeze
```
Comment (ilovezfs) [inline Library/Homebrew/rubocops/options_cop.rb]: maybe lower case f while we're here?

File: Library/Homebrew/test/rubocops/options_cop_spec.rb [hunk 2]
```diff
@@ -60,7 +60,7 @@ class Foo < Formula
         class Foo < Formula
           url 'http://example.com/foo-1.0.tgz'
           deprecated_option "examples" => "with-examples"
-          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ New Formula should not use `deprecated_option`
+          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ New Formulae should not use `deprecated_option`
```
Comment (ilovezfs) [inline Library/Homebrew/test/rubocops/options_cop_spec.rb]: ditto

File: Library/Homebrew/rubocops/options_cop.rb (hunk 1 above)
Comment (commitay) [reply Library/Homebrew/rubocops/options_cop.rb]: updated

**Comment by ilovezfs** [comment]
🚀

**Pull request merged**